supplements/eberron-rising-from-the-last-war/class-artificer.xml

```diff
@@ -824,14 +824,16 @@
 			</ul>
 			<p class="indent">You can use this energy a number of times equal to your Intelligence modifier (minimum of once), but you can do so no more than once on a turn. You regain all expended uses when you finish a long rest.</p>
 		</description>
-		<sheet>
+		<sheet usage="{{arcane jolt usage:intelligence}}/Long Rest">
 			<description>When either you hit a target with a magic weapon attack or your steel defender hits a target, you can create one of the following effects:
 			The target takes an extra {{arcane jolt damage:dice:amount}}d6 force damage.
 			Choose one creature or object you can see within 30 feet of the target restoring {{arcane jolt healing:dice:amount}}d6 hit points to it.</description>
 		</sheet>
 		<rules>
 			<stat name="arcane jolt damage:dice:amount" value="2" />
 			<stat name="arcane jolt healing:dice:amount" value="2" />
```

Comment:
it's usually "[feature]:[effect]:[details]", so stats before new ones are wrong too, you should change those to just be one stat as `"arcane jolt:dice:amount"` with value 2 and a bonus `"base"`. Then in Improved Defender feature, do the same thing and use value 4 with a bonus `"base"`.

Reply:
I _think_ I made the changes you requested. My brain is a little sluggish with a migraine